metaflow/runner/deployer.py

```diff
@@ -271,6 +271,19 @@ class DeployedFlow(object):
 
     def __init__(self, deployer: "DeployerImpl"):
         self.deployer = deployer
+        self.name = self.deployer.name
+        self.flow_name = self.deployer.flow_name
+        self.metadata = self.deployer.metadata
+
+    @staticmethod
+    def from_deployment(
+        identifier: str, metadata: str = None, type_: str = "argo-workflows"
```

Comment:
can you make this configurable through the metaflow config (and set the default to argo-workflows)

Reply:
suggestion on the metaflow config var name?

Reply:
added a TODO for now